README.md

```diff
@@ -10,7 +10,7 @@ cd votca
 mkdir build
 cd build
 cmake -DBUILD_CSGAPPS=ON -DBUILD_CTP=ON -DBUILD_XTP=ON -DCMAKE_INSTALL_PREFIX=${prefix} ..
-make -j5
+make
```

Comment:
What does it hurt to have `-j5`? We should at least say `-j2` otherwise `-j<number of cores>` would be better.

Reply:
okay that is a valid point